jobs/v3/snippets/basic_company_sample.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"golang.org/x/oauth2/google"
+	jobs "google.golang.org/api/jobs/v3"
+)
+
+const requestDeadline = 10 * time.Second	
+
+func prettyFormat(v interface{}) string {
+	j, err := json.MarshalIndent(v, "", "    ")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return string(j)
+}
+
+// [START basic_company]
+
+func makeCompany() *jobs.Company {
+	return &jobs.Company{
+		DisplayName:          "Google",
+		HeadquartersAddress:           "1600 Amphitheatre Parkway Mountain View, CA 94043",
+		ExternalId: fmt.Sprintf("company:%d", rand.New(rand.NewSource(time.Now().UnixNano())).Int63()),
+	}
+}
+
+// [END basic_company]
+
+// [START create_company]
+
+func createCompany(ctx context.Context, js *jobs.Service, toCreate *jobs.Company, parent string) (*jobs.Company, error) {
+	fmt.Println("Attempting to create a Company...")
+
+	createCtx, cancel := context.WithTimeout(ctx, requestDeadline)
+	defer cancel()
+
+	request := &jobs.CreateCompanyRequest {
+		Company: toCreate,
+	}
+
+	c, err := js.Projects.Companies.Create(parent,request).Context(createCtx).Do()
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Printf("Company created:\n %v\n", prettyFormat(c))
+	return c, nil
+}
+
+// [END create_company]
+
+// [START get_company]
+
+func getCompany(ctx context.Context, js *jobs.Service, name string) (*jobs.Company, error) {
+	fmt.Printf("Attempting to get a Company with name %s...\n", name)
+
+	getCtx, cancel := context.WithTimeout(ctx, requestDeadline)
+	defer cancel()
+
+	c, err := js.Projects.Companies.Get(name).Context(getCtx).Do()
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Printf("Company retrieved:\n %v\n", prettyFormat(c))
+	return c, nil
+}
+
+// [END get_company]
+
+// [START update_company]
+
+func patchCompany(ctx context.Context, js *jobs.Service, company *jobs.Company, parent string) (*jobs.Company, error) {
+	return patchCompanyWithFieldMask(ctx, js, company, "", parent)
```

Comment:
Is this going to end up on the documentation page? I'm afraid users won't know what this function does.